src/Core/WinSWCore/Configuration/ServiceAccount.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace winsw.Configuration
+{
+    public class ServiceAccount
+    {
+        public string name;
+        public string domain;
+        public string user;
+        public string allowservicelogon;
```

Comment:
It is optional according to https://github.com/winsw/winsw/blob/f260d7e5d95a2d50b25a48829b50392cb19c6b65/src/Core/WinSWCore/ServiceDescriptor.cs#L652